modules/tinymce/src/plugins/quickbars/test/ts/browser/CommandsTest.ts

```diff
@@ -0,0 +1,36 @@
+import { TestStore } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import QuickbarsPlugin from 'tinymce/plugins/quickbars/Plugin';
+
+describe('browser.tinymce.plugins.quickbars.CommandsTest', () => {
+  Arr.each([ true, false ], (withPresetedCommand) => {
```

Comment:
Maybe pre-defined is better?

```suggestion
  Arr.each([ true, false ], (withPredefinedCommand) => {
```

Reply:
done